modules/GT_TestGameState.php

```diff
@@ -227,11 +222,17 @@ function basicShipTiles($color, $variant)
       array_push($content, self::newContent($cargo2, 'goods', 'blue', 1, 2));
     } elseif ($variant == 2) {
       array_push($tiles, self::newTile(85, 7, 8, 0)); // double engine directly south
+      array_push($tiles, self::newTile(41, 8, 7, 90)); // cabin east
+      array_push($tiles, self::newTile(19*6, 8, 6, 0)); // purple alien life support
+      array_push($tiles, self::newTile(40, 6, 8, 180)); // cabin south west
+      array_push($tiles, self::newTile(45, 6, 7, 90)); // cabin west
       $cargo = self::newTile(58, 7, 6, 0); // hazard cargo north
       array_push($tiles, $cargo);
       array_push($tiles, self::newTile(88, 7, 5, 0)); // cannon above that
       array_push($tiles, self::newTile(3, 6, 6, 270)); // battery to west of cargo
       array_push($content, self::newContent($cargo, 'goods', 'red', 1, 1));
+    } elseif ($variant == 3) {
+      array_push($tiles, self::newTile(36, 7, 6, 0));
```

Comment:
Could you add a comment as to what this tile is?

Reply:
oh yes, sry forgot that